src/Catalyst.jl

```diff
@@ -122,7 +122,7 @@ export symmap_to_varmap
 # reaction_network macro
 include("expression_utils.jl")
 include("dsl.jl")
-export @reaction_network, @network_component, @reaction, @species
+export @reaction_network, @network_component, @reaction, @species, UndeclaredSymbolicError
```

Comment:
Why does this need to be exported?

Reply:
I thought newly-defined errors needed to be exported for some reason but will remove this.